.travis.yml

```diff
@@ -1,7 +1,34 @@
-sudo: false
-language: node_js
-node_js:
-  - 10
-  - 11
-script:
-  - npm run build && npm test
+language: rust
+sudo: required
```

Comment:
nit: `sudo` is meaningless, can be removed.

Reply:
Fixed!